packages/realm/src/List.ts

```diff
@@ -163,8 +199,49 @@ export class List<T = unknown> extends OrderedCollection<T> implements Partially
     return internal.size;
   }
 
+  /** TODO
+   * Changes the contents of the list by removing value and/or inserting new value.
```

Comment:
I am not sure how to handle multiple implementations of functions. One would hope they would inherit from one another... but they don't...